app/components/CapitalProjectsList/CapitalProjectsList.tsx

```diff
@@ -2,18 +2,25 @@ import { Box, Collapse, Flex, Text, VStack } from "@nycplanning/streetscape";
 import { Agency, CapitalProject, CapitalProjectBudgeted } from "~/gen";
 import { CapitalProjectsListItem } from "./CapitalProjectsListItem";
 import { formatFiscalYearRange } from "../../utils/utils";
+import { CapitalProjectsListNoResults } from "./CapitalProjectsListNoResults";
 
 export interface CapitalProjectsListProps {
   capitalProjects: CapitalProject[];
   agencies: Agency[];
+  capitalProjectsTotal: number;
   isExpanded: boolean;
 }
 
 export const CapitalProjectsList = ({
   capitalProjects,
   agencies,
+  capitalProjectsTotal,
   isExpanded,
 }: CapitalProjectsListProps) => {
+  if (capitalProjectsTotal === 0) {
+    return <CapitalProjectsListNoResults />;
```

Comment:
I don't really think this warrants being it's own component. You can put the contents of that component directly in this `if`.

Reply:
Done.